docs/hacking.md

```diff
@@ -37,3 +37,9 @@ $ export GOPASS_HOMEDIR=/tmp/gp1
 $ mkdir -p $GOPASS_HOMEDIR
 $ go build && ./gopass setup --crypto age --storage gitfs
 ```
+
+## Extending gopass
+
+The main extension model small binaries that use the [gopass API](https://pkg.go.dev/github.com/gopasspw/gopass/pkg/gopass/api) package. This package provides a small and easy to use API that should work with any up to date gopass setup.
```

Comment:
> The main extension model small binaries 

I find this difficult to understand, is it missing a few parts?

Reply:
Yeah, I guess so.